Makefile

```diff
@@ -4,7 +4,7 @@ all                  :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:Dss
 clean                :; forge clean
                         # Usage example: make test match=SpellIsCast
 test                 :; ./scripts/test-dssspell-forge.sh match="$(match)" block="$(block)"
-test-forge           :; ./scripts/test-dssspell-forge.sh match="$(match)" block="$(block)"
+test-forge           :; ./scripts/test-dssspell-forge.sh match="$(match)" no-match="$(no-match)" block="$(block)"
```

Comment:
Btw, we might want to remove `test-forge` as it's already for some time is the same as `test` (or at least also edit `test` command to include `no-match`

But since it's an additional improvement it's non-blocking

Reply:
Oh, I haven't noticed those commands were pointing to the same script. Will fix it right the way.